AccountCloneTest.cls

```diff
@@ -7,27 +7,28 @@ private class AccountCloneTest {
 	//Test for single record.
 	static testMethod void testSingleAccountClone() {   
 		Test.startTest();
-        	Account accountRow = new Account(Name='Test',BillingCity='TestCity');
-        	insert accountRow;
+        Account accountRow = new Account(Name='Test',BillingCity='TestCity');
+        insert accountRow;
         Test.stopTest();
         
-         Account accountList = [select Name, BillingCity from Account where id !=: accountRow.id];
+		Account accountList = [select Name, BillingCity from Account where id !=: accountRow.id];
         system.assertEquals('Test',accountList.Name); //Name should be equal to Name in accountRow
         system.assertEquals('TestCity',accountList.BillingCity); //BillingCity should be equal to BillingCity in accountRow
 	}
 	
 	//Test for bulkified data.
 	static testMethod void testBulkAccountClone() {   
+		Integer counter = 100; 
 		Test.startTest();
-        	List<Account> accountsList = new list<Account>();
-    		for (Integer i = 1; i <= 100; i++){
-          		Account acc = new Account(Name='TestAcc '+i,BillingCity='TestAccCity '+i);
-          		accountsList.add(acc);
-    		}
-    		insert accountsList;
+        List<Account> accountsList = new list<Account>();
+    	for (Integer i = 1; i <= counter; i++){
+          	Account acc = new Account(Name='TestAcc '+i,BillingCity='TestAccCity '+i);
+          	accountsList.add(acc);
+    	}
+    	insert accountsList;
     	Test.stopTest();
       	List<Account> accList = [SELECT Id FROM Account];  
-		System.assertEquals(accountsList.size(),100);
-		System.assertEquals(accList.size(),200);        
+		System.assertEquals(accountsList.size(),counter);
```

Comment:
This assert is not required.
Also as discuss earlier pls add more asserts here you are suppose to check for values of newly created accounts. see below code sample which will help you in future for writing tests.
``` java
List <Account> accList = [SELECT Id FROM Account where Id not in :accountsList];
System.assertEquals(accList.size(),counter);
Map<String, Account> accountNameToAccountMap = new Map<String, Account>();
for(Account account :accList){
  accountNameToAccountMap.put(account.Name, account);
}

for(Account oldAccount : accountsList){
  Account newAccount = accountNameToAccountMap.get(oldAccount.Name);
  System.assertEquals(newAccount.Name, oldAccount.Name);
  System.assertEquals(newAccount.BillingCity, oldAccount.BillingCity);
 // likewise you can add more asserts as per fields which you have used in your test data.
}